spec/order_spec.rb

```diff
@@ -0,0 +1,27 @@
+require './lib/order.rb'
+describe Order do
+  describe '#add_items' do
+    it 'allows to enter the dish name, quantity and total amount' do
+      expect{ subject.add_items("Pad_thai", 2, 16) }.to change { subject.my_order }.to ["Pad_thai * 2"]
+    end
+
```

Comment:
Include another test to check that adding other subsequent dishes, after the pad thais above were added, does not delete the pad thais (ie it doesn't reset the my_order each time).